apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/BrazeTools/Modal/Body.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { Button, Flex, Input } from "@ledgerhq/react-ui";
+import { useGenerateLocalBraze } from "../Hooks/useGenerateLocalBraze";
+import { useTranslation } from "react-i18next";
+
+type TabKey = "NotificationContentCard" | "ActionContentCard" | "PortfolioContentCard";
+
+export const ModalBody: React.FC = () => {
+  const { t } = useTranslation();
+  const [formData, setFormData] = useState({
+    title: "",
+    description: "",
+    image: "",
+    mainCta: "",
+    link: "",
+    secondaryCta: "",
+    cta: "",
+    url: "",
+    path: "",
+    order: undefined as number | undefined,
+  });
+
+  const { addLocalPortfolioCard, addLocalActionCard, addLocalNotificationCard, dismissLocalCards } =
+    useGenerateLocalBraze();
+
+  const handleAddCard = () => {
+    const { title, description, image, mainCta, link, secondaryCta, cta, url, path, order } =
+      formData;
+    if (selectedTab === "PortfolioContentCard") {
+      addLocalPortfolioCard(title, description, image, order);
+    } else if (selectedTab === "ActionContentCard") {
+      addLocalActionCard(title, description, image, mainCta, link, secondaryCta, order);
+    } else if (selectedTab === "NotificationContentCard") {
+      addLocalNotificationCard(title, description, cta, false, url, path, order);
+    }
+  };
+
+  const handleChange = (field: string, value: string | number | boolean) => {
+    setFormData(prev => ({ ...prev, [field]: value }));
+  };
+
+  const tabs: { key: TabKey; label: string }[] = [
+    { key: "NotificationContentCard", label: "Notification Content Card" },
+    { key: "ActionContentCard", label: "Action Content Card" },
+    { key: "PortfolioContentCard", label: "Portfolio Content Card" },
+  ];
+
+  const [selectedTab, setSelectedTab] = useState<TabKey>(tabs[0].key);
+
+  const inputFields: Record<TabKey, { field: string; placeholder: string }[]> = {
+    PortfolioContentCard: [{ field: "image", placeholder: "Image URL" }],
+    ActionContentCard: [
+      { field: "image", placeholder: "Image URL" },
+      { field: "mainCta", placeholder: "Main CTA" },
+      { field: "link", placeholder: "Link" },
+      { field: "secondaryCta", placeholder: "Secondary CTA" },
+    ],
+    NotificationContentCard: [
+      { field: "cta", placeholder: "CTA" },
+      { field: "url", placeholder: "URL" },
+      { field: "path", placeholder: "Path" },
+    ],
+  };
+
+  return (
+    <Flex flexDirection="column" rowGap={24}>
+      <Flex flexDirection="row" columnGap={24}>
+        {tabs.map(tab => (
+          <Button
+            flex={1}
+            variant={selectedTab === tab.key ? "color" : "shade"}
+            key={tab.key}
+            onClick={() => setSelectedTab(tab.key)}
+          >
+            {tab.label}
+          </Button>
+        ))}
+      </Flex>
+      <Input value={formData.title} onChange={e => handleChange("title", e)} placeholder="Title" />
```

Comment:
Can we put the name of the fields in front of them (on the same row) so we know which are which once we start filling them ?

Reply:
![image](https://github.com/user-attachments/assets/38e26fe7-a184-4e08-8c64-14065905cc57)
Something like that ? I need to set min-width for inputs because it's wrapped inside a display block so flex:1 doesn't work